### ISSUE TITLE
ctrl/cmd-N doesn't start new chat

### ISSUE THREAD

**Issue by terinjokes** [Sytten/Facebook-Messenger-Desktop#10]
Not sure where you would want this logic (it's not quite window behavior…)

``` javascript
document.querySelector('a[href="/new"]').dispatchEvent(
  new MouseEvent('click', {view:window, bubbles:true, cancelable:true})
);
```

**Comment by Sytten** [comment]
I'm not sure to see what its supposed to do... On my computer the ctrl+N just open a new FF window on messenger.com and doesnt do anything on the app.
The code is already pretty messy (I looked at the new version, its much better), so I'm not sure where to put that...

**Comment by terinjokes** [comment]
It's a feature request. In this app we should make it open a new
conversation.

**Comment by Sytten** [comment]
Yes thats a good idea, we should do a new file for shortcut handling. Anyway I need to intercept the ESC key, so that would be an idea.
